docs/markdown/Using Pants/concepts/enabling-backends.md

```diff
@@ -16,43 +16,96 @@ backend_packages = [
+| `pants.backend.python.lint.pydocstyle`               | Enables pydocstyle, a Python docstring convention checker: <https://www.pydocstyle.org/>           | [Linters and formatters](doc:python-linters-and-formatters)       |
+| `pants.backend.python.lint.pylint`                   | Enables Pylint, the Python linter: <https://www.pylint.org>                                        | [Linters and formatters](doc:python-linters-and-formatters)       |
+| `pants.backend.python.lint.pyupgrade`                | Enables Pyupgrade, which upgrades to new Python syntax: <https://pypi.org/project/pyupgrade/>      | [Linters and formatters](doc:python-linters-and-formatters)       |
+| `pants.backend.python.lint.yapf`                     | Enables Yapf, the Python formatter: <https://pypi.org/project/yapf/>                               | [Linters and formatters](doc:python-linters-and-formatters)       |
+| `pants.backend.python.typecheck.mypy`                | Enables MyPy, the Python type checker: <https://mypy.readthedocs.io/en/stable/>.                   | [typecheck](doc:python-check-goal)                                |
+| `pants.backend.shell`                                | Core Shell support, including shUnit2 test runner.                                                 | [Shell overview](doc:shell)                                       |
+| `pants.backend.shell.lint.shfmt`                     | Enables shfmt, a Shell autoformatter: <https://github.com/mvdan/sh>.                               | [Shell overview](doc:shell)                                       |
+| `pants.backend.shell.lint.shellcheck`                | Enables Shellcheck, a Shell linter: <https://www.shellcheck.net/>.                                 | [Shell overview](doc:shell)                                       |
+| `pants.backend.tools.preamble`                       | Enables "preamble", a Pants fixer for copyright headers and shebang lines                          | [`preamble`](doc:reference-preamble)                              |
+| `pants.backend.tools.taplo`                          | Enables Taplo, a TOML autoformatter: <https://taplo.tamasfe.dev>                                   |                                                                   |
+| `pants.backend.url_handlers.s3`                      | Enables accessing s3 via credentials in `file(source=http_source(...))`                            |                                                              |
+
+Available experimental backends
+-------------------------------
+
+Pants supports numerous extra backends that aren't as stable as the backends above, due to less extensive documentation, testing and sometimes missing features that inhibit usability. If any of these backends are relevant to you, please try them, [ask any questions](doc:getting-help) you have, and [contribute improvements](doc:contributor-overview)!
```

Comment:
I've just made up a bit of a blurb here about why things are might experimental. I'm hoping we can avoid this PR being stuck on it, since it's pretty generic. Do we have an existing policy around experimental/unstable-ness?

Reply:
I think you covered it pretty well :)

Reply:
This was a thing: https://blog.pantsbuild.org/quick-feedback-on-new-features-via-experimental-backends/

Reply:
@thejcannon such a Stu comment! 😂

Reply:
Nice one, I'll see if I should adjust this paragraph and will include a link to the post too.